SConstruct

```diff
@@ -1,36 +1,36 @@
 # Preliminaries
 import os
 import sys
-sys.dont_write_bytecode = True # http://stackoverflow.com/questions/154443/how-to-avoid-pyc-files
+sys.dont_write_bytecode = True # Don't write .pyc files
 
 # Test for proper prerequisites and setup
 from configuration_test import configuration_test
-[mode, sf, cache_dir] = configuration_test(ARGUMENTS)
-import gslab_scons 
+[mode, sf, cache_dir] = configuration_test(ARGUMENTS, 
+                                           gslab_python_version = '3.0.5')
```

Comment:
We will. Unless you can think of a downside of doing so, I'll update all of the version numbers in gslab-econ/gslab_python#66's branch to 4.0.0.